### DISCUSSION TITLE
[Bug]: JsDocs @deprecated tag does not show description

### DISCUSSION THREAD

**Discussion by miszol1** [storybookjs/storybook#26154] [Help]
### Describe the bug

When @ deprecated tag is used to describe input property, description is not displayed in Docs tab.
![image](https://github.com/storybookjs/storybook/assets/24455637/af2244c6-0dcf-4b49-bbbf-1ef8ace01efb)
![image](https://github.com/storybookjs/storybook/assets/24455637/63fe32b0-50ec-4a53-b9b7-07f3b29042c9)

### To Reproduce

https://stackblitz.com/edit/github-6gtqqv?file=src%2Fstories%2Fbutton.component.ts

### System

```bash
Storybook Environment Info:

  System:
    OS: Linux 5.0 undefined
    CPU: (8) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Shell: 1.0 - /bin/jsh
  Binaries:
    Node: 18.18.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 10.2.3 - /usr/local/bin/npm <----- active
    pnpm: 8.14.0 - /usr/local/bin/pnpm
```

**Comment by vanessayuenn** [comment]
I'm not sure if there is a standard defined by JsDoc, but if you put the description before the `@deprecated` line, it shows up in docs.

```
  /**
   * Is this the principal call to action on the page?
   * @deprecated qwe
   */
  @Input()
  primary = false;
```

Even if it is a bug though, I think it would be on compodoc side.

**Reply by miszol1** [reply]
@vanessayuenn Description is displayed, but what about deprecated description? It worked in previous versions of storybook so for me it's a regression. In all stories in which use @ deprecated tag, description disappeard with new version :(
![image](https://github.com/storybookjs/storybook/assets/24455637/2cc3aea5-bab7-4ead-a198-76c8f5825ce9)
In documentation.json you can find that @ deprecated tag was read properly by compodoc, but it is not displayed in SB docs.

Also, will new tags or line distinguishing functionality be added eventually?

**Reply by vanessayuenn** [reply]
alright, i reopened the bug #26171.

> Also, will new tags or line distinguishing functionality be added eventually?

It's not in our roadmap currently, but we will gladly accept a PR implementing this!
